sqlite.cpp

```diff
@@ -169,14 +355,12 @@ bool SQLite::query_with_args(String query, Array args) {
 	sqlite3_stmt *stmt = prepare(query.utf8().get_data());
 
 	// Failed to prepare the query
-	if (!stmt) {
-		return false;
-	}
+	ERR_FAIL_COND_V_MSG(stmt == nullptr, false, "SQLiteQuery preparation error: " + get_last_error_message());
```

Comment:
Review with peers mentioned that the error message is not "SQLiteQuery" specific, so it should be just "Query".